client/pool.c

```diff
@@ -240,10 +240,6 @@ int xdag_initialize_pool(const char *pool_arg)
 		return -1;
 	}
 
-	xdag_mess("Starting general mining thread...");
-
-	g_stop_general_mining = 0;
```

Comment:
Marco, you removed this line and pool never generates blocks now.

Reply:
yes, #432